docs/DeveloperGuide.adoc

```diff
@@ -153,94 +153,234 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-// tag::undoredo[]
-=== [Proposed] Undo/Redo feature
-==== Proposed Implementation
+=== Delivered feature
+The delivered feature allows the user to mark orders as delivered after delivering an order.
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
-Additionally, it implements the following operations:
+The delivered feature was implemented as a `DoneCommand` in the `Logic` package. +
 
-* `VersionedAddressBook#commit()` -- Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` -- Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` -- Restores a previously undone address book state from its history.
+==== Class Diagram for Done feature
+image::DoneClassDiagram.svg[]
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()` and `Model#redoAddressBook()` respectively.
+==== Sequence Diagram for delivering an order
+The sequence diagram for the *Delivered Command* is shown below.
 
-Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
+image::DoneSequenceDiagram.svg[]
```

Comment:
I think non standardized UML notation might need to be disabled. You can refer to this link: https://github.com/nus-cs2103-AY1920S2/forum/issues/88